source/PhysicalQuantity/Force.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Force extends AbstractPhysicalQuantity
+{
+	use HasSIUnitsTrait;
+
+	protected static $unitDefinitions;
+
+	protected static function initialize()
+	{
+
+		// newton
+		$newton = UnitOfMeasure::nativeUnitFactory('N');
+		$newton->addAlias('newton');
+		static::addUnit($newton);
```

Comment:
we've been trying to include plurals as well, please add another alias  'newtons' after the 'newton' one here.